e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -523,23 +516,20 @@ describeEE("scenarios > embedding > full app", () => {
         },
       });
 
+      cy.get("@postMessage").invoke("resetHistory");
       cy.findByTestId("app-bar").findByText("Our analytics").click();
 
       cy.findByRole("heading", { name: "Metabase analytics" }).should(
         "be.visible",
       );
-      cy.get("@postMessage").then(postMessage => {
-        expect(
-          postMessage.lastCall.calledWith({
-            metabase: {
-              type: "frame",
-              frame: {
-                mode: "fit",
-                height: 800,
-              },
-            },
-          }),
-        ).to.be.true;
+      cy.get("@postMessage").should("have.been.calledWith", {
```

Comment:
Just change this to a standard assertion for consistency. This works great since I now reset the spy before changing the dashboard height.